src/Wishlist.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+import DiveWish from './DiveWish.js';
+
+export default function Wishlist({ diveWishes }) {
+  return (
+    <Wrapper>
+      {diveWishes.length > 0 ? (
+        diveWishes.map((wish, index) => (
+          <DiveWish
+            key={index}
```

Comment:
The `index` as key is not a good solution. The index of an element can change and then React might cause render problems. Especially since you add a new item at the start of the array. How about using `nanoid` to create a new `id` for each item? https://github.com/ai/nanoid

Reply:
Ok, will do that. 
Btw I just saw in the documentation that the use of nanoid in React is not recommended as it isn't consistent among renders... it says to use the index, even though it is not perfect.

Reply:
Interesting! Do you have a link to an article or so?